buildspec.yml

```diff
@@ -60,15 +64,19 @@ phases:
       - cp -r docker/build_artifacts/* $build_dir/
       - CPU_PY3_TAG="$FRAMEWORK_VERSION-cpu-py3-$BUILD_ID"
       - GPU_PY3_TAG="$FRAMEWORK_VERSION-gpu-py3-$BUILD_ID"
+      - EIA_PY3_TAG="$EIA_FRAMEWORK_VERSION-eia-py3-$BUILD_ID"
       - docker build -f "$build_dir/$cpu_dockerfile" -t $PREPROD_IMAGE:$CPU_PY3_TAG $build_dir
       - docker build -f "$build_dir/$gpu_dockerfile" -t $PREPROD_IMAGE:$GPU_PY3_TAG $build_dir
+      # PY2 not offered for EIA PyTorch
+      - docker build -f "$build_dir/$eia_dockerfile" -t $PREPROD_IMAGE:$EIA_PY3_TAG $build_dir
```

Comment:
Cannot use `$build_dir` as is, because it points to the `docker/1.4.0` directory.

Please set it to `build_dir="docker/$EIA_FRAMEWORK_VERSION/py$EIA_PY3_VERSION"`

Reply:
Good catch